lib/python/rose/config.py

```diff
@@ -1079,7 +1080,7 @@ def dump(self, root, target=sys.stdout, sort_sections=None,
             else:
                 section_keys.append(key)
         if root_option_keys:
-            handle.write(blank)
+            self._write_safely(handle, blank)
```

Comment:
Is it worth making this a module method, or just leaving it as a `@staticmethod` of this class?